docs/pages/tutorial/onchain-chess/1-action.md

```diff
@@ -45,15 +37,14 @@ To play chess, you need, to start game, spawn the pieces, and move around the bo
 
 Should be noted that `actions` is the contract name.
 
-3. Write a `spawn` function that accepts the `white address`, and `black address` as input and set necessary states using `set!(...)`. Implement the `player` entity from player model. Implement the game entity, comprised of the `Game` model and `GameTurn` model we created in the `game.cairo` and implement the piece entities from a1 to h8 containing the correct `PieceType` in the `spawn` fn.
+3. Write a `spawn` function that accepts the `white address`, and `black address` as input and set necessary states using `set!(...)`. Implement the `player` entity from player model. Implement the game entity, comprised of the `Game` model and `GameTurn` model we created in the `game.cairo` and implement the piece entities from a1 to h8 containing the correct `PieceType` in the `spawn` fn. Paste this code below inside `mod actions`.
```

Comment:
The term `implement` with an entity can sound strange at first. Do you think we can rephrase a bit this one to make more clear to the user that an entity is at the end an ID (or combination of IDs) that identifies it?

Reply:
Sure I am gonna fix it. 
By the way, I messed between my branches. I will apply those changes on this [PR](https://github.com/dojoengine/book/pull/242)